docs/guides/cloud/integrations/bitbucket-integration.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Bitbucket Integration
-sidebar_position: 90
+sidebar_position: 10
```

Comment:
we should sort these nav items the same order they were in the initial nav. IE, GitHub, GitLab, Bitbucket, etc...

Reply:
I set them per the chat about analytics for these pages